FarihaIS/VanRoomies
# feat (nav bar, profile, listings): Profile and Listings Fragments + Bottom Navigation Bar

# Welcome to VanRoomies! 👋✈️

Fixes: #16 & #27 

## Description of the change:
This commit includes three main features: 
- The fragment containing the Profile View where users can see (and soon, modify) their bio, description, profile picture, and preferences
- The fragment containing the Listings View where house listings are shown in cards
- A bottom navigation bar to cleanly move from one fragment to another (done in paired programming with @FarihaIS 

## How to manually test:
1. Gradle Sync if needed
2. Build the project
3. Run the app in a Pixel 3 Emulator
4. Login if not already done
5. After login (or on startup if already logged in), the profile fragment should be shown first
